src/main/java/org/gridsuite/study/server/repository/StudyEntity.java

```diff
@@ -129,6 +129,9 @@ public class StudyEntity extends AbstractManuallyAssignedIdentifierEntity<UUID>
         ))
     private List<NodeAliasEmbeddable> nodeAliases;
 
+    @Column(name = "mono_root", columnDefinition = "boolean default true")
+    private boolean monoRoot;
```

Comment:
```suggestion
    @Column(name = "mono_root")
    private boolean monoRoot = true;
```

then deal with default value in database with liquibase changelog

Reply:
The default value depends on the number of root networks in the study. Based on this, we can determine the flag’s state (true or false) that I did with update changeset , no need to have default value I think *

Reply:
already discussed